### ISSUE TITLE
frontal lisp misgenders women

### ISSUE THREAD

**Issue by Emisse** [space-wizards/space-station-14#32030]
this has caused an administration issue 
![image](https://github.com/user-attachments/assets/ae953208-7d08-434c-9c7c-bd447676f77a)
![image](https://github.com/user-attachments/assets/95a2d94d-5689-4bb2-b420-40bbe20befdf)

**Comment by Emisse** [comment]
owo accent also misgenders women 
![image](https://github.com/user-attachments/assets/d31daaa3-844f-4f8b-a635-d2c42170ab66)
![image](https://github.com/user-attachments/assets/06887ebd-b8df-453d-b209-e4889c9ed8ed)
bothing of these accents combined make it impossible to use correct pronouns for she/her

**Comment by bruhmogus** [comment]
isn't lisp SUPPOSED to cause problems by making all s letters into a th

**Comment by dragonryan06** [comment]
Yeah this isn't a bug, it's not like swapping the pronoun its just doing a regex match for a certain consonant and swapping it for a different consonant sound to make it look like a speech impediment sounded out.

Is it also a bug that its "impossible" to say "know" as a dwarf because it will always be swapped for "ken?" no. This is working exactly as intended.

**Comment by beck-thompson** [comment]
I agree, this is just how lisps work! If this is becomes a massive issues we can bring it up again just let me know
